test/edu-benefits/pages/schoolSelection.unit.spec.jsx

```diff
@@ -1,141 +1,76 @@
+import _ from 'lodash/fp';
+
 import React from 'react';
 import { findDOMNode } from 'react-dom';
 import { expect } from 'chai';
 import sinon from 'sinon';
 import ReactTestUtils from 'react-addons-test-utils';
 
 import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
-import formConfig from '../../../src/js/edu-benefits/pages/schoolSelection.js';
```

Comment:
That makes sense to me given how you've organized the config for this page for the different forms (though like I said above, I'm not sure how I feel about that).